docs/api/index.rst

```diff
@@ -11,6 +11,12 @@ API
 Changelog
 ---------
 
+1.19 (unreleased)
+'''''''''''''''''
+
+- New `contains_` and `contains_any` filter operators (fixes #343)
+
```

Comment:
Could you please add them to https://github.com/Kinto/kinto/blob/master/docs/api/1.x/filtering.rst ?

Reply:
Yes @jlebunetel is working on the documentation part :+1: